sopel/modules/ipython.py

```diff
@@ -39,6 +39,13 @@
 console = None
 
 
+def shutdown(bot):
+    try:
+        del bot.memory['iconsole_running']
+    except KeyError:
+        pass
```

Comment:
Worth looking into whether the console actually stops if the module is unloaded (and then reloaded, or not) while the console is active. If it's not stopped, then this should either not remove the "running" flag from memory, or it should terminate the console at `shutdown()`.

Reply:
The console doesn't stop, so keeping the "running" flag around instead (no `shutdown()`).